Composer/packages/client/src/components/ProjectTree/TriggerCreationModal.tsx

```diff
@@ -194,52 +199,67 @@ export interface LuFilePayload {
   content: string;
 }
 
+export interface LgFilePayload {
+  id: string;
+  lgTemplates: LgTemplate[];
+}
+
+export interface QnAFilePayload {
+  id: string;
+  content: string;
+}
+
 // -------------------- TriggerCreationModal -------------------- //
 
 interface TriggerCreationModalProps {
   dialogId: string;
   isOpen: boolean;
   onDismiss: () => void;
-  onSubmit: (dialog: DialogInfo, intent?: LuIntentSection) => void;
+  onSubmit: (dialogId: string, formData: TriggerFormData) => void;
 }
 
 export const TriggerCreationModal: React.FC<TriggerCreationModalProps> = (props) => {
   const { isOpen, onDismiss, onSubmit, dialogId } = props;
   const dialogs = useRecoilValue(validatedDialogsSelector);
 
   const projectId = useRecoilValue(projectIdState);
-  const schemas = useRecoilValue(schemasState);
   const userSettings = useRecoilValue(userSettingsState);
-
   const dialogFile = dialogs.find((dialog) => dialog.id === dialogId);
   const isRegEx = (dialogFile?.content?.recognizer?.$kind ?? '') === regexRecognizerKey;
+  const recognizer = get(dialogFile, 'content.recognizer', '');
+  const isLUISnQnA = typeof recognizer === 'string' && recognizer.endsWith('.lu.qna');
   const regexIntents = dialogFile?.content?.recognizer?.intents ?? [];
-  const isNone = !dialogFile?.content?.recognizer;
   const initialFormData: TriggerFormData = {
     errors: initialFormDataErrors,
-    $kind: isNone ? '' : intentTypeKey,
+    $kind: intentTypeKey,
     event: '',
     intent: '',
     triggerPhrases: '',
     regEx: '',
   };
   const [formData, setFormData] = useState(initialFormData);
-  const [selectedType, setSelectedType] = useState(isNone ? '' : intentTypeKey);
+  const [selectedType, setSelectedType] = useState(intentTypeKey);
   const showIntentName = selectedType === intentTypeKey;
   const showRegExDropDown = selectedType === intentTypeKey && isRegEx;
-  const showTriggerPhrase = selectedType === intentTypeKey && !isRegEx;
+  const showTriggerPhrase = selectedType === intentTypeKey && isLUISnQnA;
```

Comment:
Covered the `.lu.qna` case but where is the `.lu` case?

Reply:
These is no .lu anymore.